### DISCUSSION TITLE
macos nc help

### DISCUSSION THREAD

**Discussion by luqff** [physics-data/faq#97] [Q&A] [answered]
macOS 使用nc -N命令不兼容，导致无法完成saas作业

**Comment by heroxbd** [comment, marked as answer]
macOS 的 nc 没有 `-N` 的选项，尝试修改 `saas.sh` 中的 `nc -N` 的选项再试。

如果是 GNU netcat ，把它换成 `nc -l -p 1080`


netcat 版本的判断方法

`nc -help` 查看第一行，

如果出现 `[v1.10-46]` 说明是经典版本的 netcat

如果出现 `OpenBSD netcat (Debian patchlevel 1.217-3)`  

如果出现 `GNU` 字样则为 GNU netcat

**Reply by heroxbd** [reply]
经过试验，`brew install netcat` 得到的 nc 也不支持 `-N` 的选项。

**Reply by heroxbd** [reply]
brew 得到的是 GNU netcat
